addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -1093,7 +1093,8 @@ test("out-of-focus notif on needaction message in channel", async () => {
             }
         },
     });
-    onRpcBefore("/mail/action", async (args) => {
+    mockService("mail.out_of_focus", { setInterval: () => {} }); // so that no setInterval runs
```

Comment:
Why do we need to do that?

Reply:
Pretty sure this was an issue when the out-of-focus code had blinking effect, which was removed recently, thus the `mockService({ setInterval })` no longer makes sense